src/Framework/MockObject/InvocationMocker.php

```diff
@@ -93,7 +93,7 @@ public function expects(MatcherInvocation $matcher): BuilderInvocationMocker
         return new BuilderInvocationMocker(
             $this,
             $matcher,
-            $this->configurableMethods
+            ...$this->configurableMethods
```

Comment:
Argument 3 of PHPUnit\Framework\MockObject\Builder\InvocationMocker::__construct expects PHPUnit\Framework\MockObject\ConfigurableMethod, string provided
```
            ...$this->configurableMethods
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
```